baselines/agents/simple_agents.py

```diff
@@ -135,20 +129,15 @@ def get_agent_cls(agent_class_name):
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument(
-        "--task-config", type=str, default="tasks/pointnav.yaml"
-    )
+    parser.add_argument("--success_distance", type=float, default=0.2)
     parser.add_argument("--agent_class", type=str, default="GoalFollower")
     args = parser.parse_args()
 
     agent = get_agent_cls(args.agent_class)(
-        habitat.get_config(args.task_config)
+        success_distance=args.success_distance
     )
-    benchmark = habitat.Benchmark(args.task_config)
-    metrics = benchmark.evaluate(agent)
-
-    for k, v in metrics.items():
-        habitat.logger.info("{}: {:.3f}".format(k, v))
+    challenge = habitat.Challenge()
```

Comment:
I suggest we retain the old `benchmark` usage here instead of `challenge`. This code is orthogonal to challenge codebase I think, also if someone is testing locally `benchmark` fits in better semantically compared to `challenge`